test/gtest/common/test.cc

```diff
@@ -41,6 +41,14 @@ test_base::~test_base() {
                        m_state == NEW ||    /* can be skipped from a class constructor */
                        m_state == ABORTED,
                        "state=%d", m_state);
+
+    ucs_mem_info_t info;
+    ucs_get_mem_info(NULL, 0, &info);
+    /* make sure no DONTFORK regions left behing */
+    if (info.flags & UCS_BIT(UCS_MEM_FLAG_DONTCOPY)) {
+        madvise((void *)info.dc_range_start,
+                info.dc_range_end - info.dc_range_start, MADV_DOFORK);
+    }
```

Comment:
can madvise fail if there are multiple DC regions with a hole in between?
seems ucs_get_mem_info should have callback function for regions

Reply:
it's ok to have holes